PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -122,16 +128,13 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
 run2_nanoAOD_92X.toModify(ptRatioRelForEle, srcLep = "slimmedElectronsUpdated")
 
 import EgammaAnalysis.ElectronTools.calibratedElectronsRun2_cfi
-calibratedPatElectrons80X = EgammaAnalysis.ElectronTools.calibratedElectronsRun2_cfi.calibratedPatElectrons.clone(
-    electrons = cms.InputTag("slimmedElectronsUpdated"),
-    correctionFile = cms.string("PhysicsTools/NanoAOD/data/80X_ichepV1_2016_ele"),
-    semiDeterministic = cms.bool(True)
-)
-energyCorrForEle80X =  cms.EDProducer("ElectronEnergyVarProducer",
-    srcRaw = cms.InputTag("slimmedElectronsUpdated"),
-    srcCorr = cms.InputTag("calibratedPatElectrons80X"),
-)
+
 import RecoEgamma.EgammaTools.calibratedEgammas_cff
+
+calibratedPatElectrons80XLegacy = RecoEgamma.EgammaTools.calibratedEgammas_cff.calibratedPatElectrons.clone(
```

Comment:
Please announce the change to the calibration function next time, when opening the PR! It is a delicate change that needs care in the validation.

Reply:
ah yes. you're right I should have mentioned that I changed the energy scale, sorry. Again just not been with it this week. Incidentally the old one was incorrect (its for prompt), it is a needed change.